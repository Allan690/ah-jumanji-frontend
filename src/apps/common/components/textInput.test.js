import React from "react";

import {shallow} from "enzyme";

import Input from "./textInput";


describe("Header Section", () => {
  const component = shallow(<Input
    debug label="label" value="value" placeholder="value" onChange={() => {
  }} type="text" name="value"
  />);

  it("should render correctly ", () => {
    expect(component).toMatchSnapshot();
  });
});
