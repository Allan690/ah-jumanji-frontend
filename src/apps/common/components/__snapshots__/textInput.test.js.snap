// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Section should render correctly  1`] = `
<div>
  <div
    className="form-group"
  >
    label
    <input
      className="form-control"
      id="value"
      onChange={[Function]}
      placeholder="value"
      required=""
      type="text"
      value="value"
    />
  </div>
</div>
`;
